test/e2e/suite/nodeadm/nodeadm_test.go

```diff
@@ -154,7 +156,10 @@ var _ = Describe("Hybrid Nodes", func() {
 						test.Logger.Info("Skipping nodeadm uninstall from the hybrid node...")
 						return
 					}
-					Expect(test.NewCleanNode(provider, testNode.PeerdNode().Name, testNode.PeerdNode().Instance.IP).Run(ctx)).To(
+
+					n := testNode.PeerdNode()
+					cleanNode := test.NewCleanNode(provider, testNode.PeeredNode.CreateNodeInfrastrastructureCleaner(n.Instance.ID), n.Name, n.Instance.IP)
```

Comment:
is the cleanode the only place we run this infracleaner?  in the case of conformance we dont call clean node, we just do the normal cleanup in NodeCleanup.Cleanup.